apps/server/src/shared/domain/interface/video-conference-scope.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum VideoConferenceScope {
+	BOARD = 'board',
```

Comment:
AFAIK we wanted to attach a video conference to a board element. So shouldn't the scope then be the element and not the board?